src/strategies/api/examples.json

```diff
@@ -5,8 +5,6 @@
       "name": "api",
       "params": {
         "api": "https://gateway.pinata.cloud/ipfs",
-        "symbol": "LIFE",
-        "decimals": 0,
```

Comment:
Awesome thanks @ori-wagmi but we still use decimals here, maybe we can make it optional? (default to `0`) https://github.com/snapshot-labs/snapshot-strategies/blob/d0c8ba48bda1e6e77be09f83bc18316b65edcff6/src/strategies/api/index.ts#L44

Reply:
- i continued to removed decimals from example.json
- made code to default to 0
- updated readme to add a "decimals" section